openapi/tests/openapi_integration/helpers/collection_setup.py

```diff
@@ -1,4 +1,3 @@
-import os
```

Comment:
I thought you fixed all those in a dedicated PR.
Do you a rebase maybe here?

Reply:
🤦 rebasing